engine/apps/api/tests/test_schedules.py

```diff
@@ -1103,7 +1103,7 @@ def test_merging_same_shift_events(
     )
     now = timezone.now().replace(hour=0, minute=0, second=0, microsecond=0)
     start_date = now - timezone.timedelta(days=7)
-    request_date = start_date
+    request_date = start_date + timezone.timedelta(days=1)
```

Comment:
why do we suddenly need to add 1 day to this?

Reply:
This test has been failing a few times in CI, I'm guessing it could be related to the fact that the shift is setup for a couple of hours starting on `start_date`, and we were using that date for the request, potentially leaving the shift off during filtering (depending on timezones, etc). So moving the request date one day later I expect we can get a consistent output for this.

Reply:
oh nice, good idea! 🤞